docs/bokeh/source/docs/user_guide/annotations.rst

```diff
@@ -178,7 +178,7 @@ Manual legends
 To build a legend by hand, don't use any of the ``legend`` arguments and instead
 assign values to the various properties of a |Legend| object directly.
 
-See :bokeh-tree:`examples/models/file/legends.py` for an example.
+See :bokeh-tree:`examples/file/legends.py` for an example.
```

Comment:
```suggestion
See :bokeh-tree:`examples/models/legends.py` for an example.
```
Looks like the wrong folder was deleted.

Reply:
Thanks for the catch @mosc9575 !